### PR TITLE
Related content

### PR DESCRIPTION
**Pull request by thomasridd**
### What

Add related content to census dataset landing page

### How to review

Run tests
Sense check
Run locally (sorry for that)

### Who can review

@mr-nick17

### CODE REVIEW

File: assets/templates/partials/census/related-content.tmpl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<section id="related-content" aria-label="{{ localise "RelatedContentTitle" .Language 1 }}">
+    <h2 class="ons-u-mt-xl ons-u-pb-no ons-u-pt-no">{{ localise "RelatedContentTitle" .Language 1 }}</h2>
+    {{ range $i, $item := .DatasetLandingPage.RelatedContentItems }}
```
Comment (mr-nick17) [inline assets/templates/partials/census/related-content.tmpl:3]: You've assigned the index var `$i` but not used it. Could just do a (simpler) range 
```suggestion
    {{ range .DatasetLandingPage.RelatedContentItems }}
```

File: assets/templates/partials/census/related-content.tmpl [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        <p>
+            <a href="{{ $item.Link }}">{{ $item.Title }}</a>
```
Comment (mr-nick17) [inline assets/templates/partials/census/related-content.tmpl:5]: If doing the above, these should become something like
```suggestion
            <a href="{{ .Link }}">{{ .Title }}</a>    
```

File: assets/templates/partials/census/related-content.tmpl [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<section id="related-content" aria-label="{{ localise "RelatedContentTitle" .Language 1 }}">
+    <h2 class="ons-u-mt-xl ons-u-pb-no ons-u-pt-no">{{ localise "RelatedContentTitle" .Language 1 }}</h2>
+    {{ range $i, $item := .DatasetLandingPage.RelatedContentItems }}
+        <p>
+            <a href="{{ $item.Link }}">{{ $item.Title }}</a>    
+            <br/>
+            {{ $item.Text }}
```
Comment (mr-nick17) [inline assets/templates/partials/census/related-content.tmpl:7]:
```suggestion
            {{ .Text }}
```

**Review by mr-nick17 — COMMENTED** [review]
Very minor comment, looks good to me

File: assets/templates/partials/census/related-content.tmpl (hunk 1 above)
Comment (thomasridd) [reply assets/templates/partials/census/related-content.tmpl]: Thanks for suggestion - I've updated

File: mapper/census.go [hunk 4]
```diff
@@ -291,6 +291,17 @@ func CreateCensusDatasetLandingPage(ctx context.Context, req *http.Request, base
...
+	p.DatasetLandingPage.RelatedContentItems = []datasetLandingPageCensus.RelatedContentItem{}
+	if d.RelatedContent != nil {
+		for _, content := range *d.RelatedContent {
+			p.DatasetLandingPage.RelatedContentItems = append(p.DatasetLandingPage.RelatedContentItems, datasetLandingPageCensus.RelatedContentItem{
+				Title: content.Title,
+				Link:  content.HRef,
+				Text:  content.Description,
+			})
+		}
+	}
```
Comment (mr-nick17) [inline mapper/census.go:303]: Does this need to be added to the 'table of contents' too? 

i.e. `p.TableOfContents...`

**Review by mr-nick17 — CHANGES_REQUESTED** [review]
Not been able to test in mongo locally, one last suggestion/thought. Otherwise LGTM

**Review by mr-nick17 — APPROVED** [review]

**Pull request merged**